src/domain/storage/storage-bucket/storage.bucket.service.ts

```diff
@@ -260,23 +260,29 @@ export class StorageBucketService {
     }
   }
 
-  public async addDocumentToBucketOrFail(
-    storageBucketId: string,
+  public async addDocumentToStorageBucketOrFail(
```

Comment:
I needed a function like `addDocumentToBucketOrFail` but passing the bucket not just the id, because my bucket is not yet saved to database here